### ISSUE TITLE
Add list of live streams

### ISSUE THREAD

**Issue by janisozaur** [Limetric/OpenRCT2.org#37]
According to BlindIRL on Twitch, there used to be a "live streams" element on the "old website", presumably openrct.net. This drove a lot of users to the streams, especially the multiplayer mondays.

I would like to request adding such element to the website in order to promote such streams. It could use streams listed at https://www.twitch.tv/directory/game/RollerCoaster%20Tycoon%202

**Comment by JarnoVgr** [comment]
Yes the former .net site had the stream overview. I'll add one soon! :) 




On Mon, Nov 19, 2018 at 10:05 PM +0100, "Michał Janiszewski" <notifications@github.com> wrote:












According to BlindIRL on Twitch, there used to be a "live streams" element on the "old website", presumably openrct.net. This drove a lot of users to the streams, especially the multiplayer mondays.


I would like to request adding such element to the website in order to promote such streams. It could use streams listed at https://www.twitch.tv/directory/game/RollerCoaster%20Tycoon%202
